TicketBooking/src/lsp/corrections/Travel.java

```diff
@@ -0,0 +1,10 @@
+package lsp.corrections;
+
+public interface Travel {
+    public void search(String from, String to);
+    public void book(String from, String to);
+
+    public void cancel(String from, String to);
```

Comment:
all the methods in public interface are by default public not need to mention public access modifier